xbmc/filesystem/PluginDirectory.cpp

```diff
@@ -197,7 +197,7 @@ bool CPluginDirectory::GetPluginResult(const std::string& strPath, CFileItem &re
     resultItem.SetDynPath(newDir.m_fileResult->GetPath());
     resultItem.SetMimeType(newDir.m_fileResult->GetMimeType());
     resultItem.SetContentLookup(newDir.m_fileResult->ContentLookup());
-    resultItem.UpdateInfo(*newDir.m_fileResult);
+    resultItem.UpdateInfo(*newDir.m_fileResult, true, false);
```

Comment:
Grrr, I just don't like those bools. I would have said create another function called `MergeInfo` that only took the file item argument (internally it can call UpdateInfo) and have your `Merge` function private.

Then it is very clear what happens here. But I also know that VideoTag is special so someone might comment on my suggestion.